src/domain/templates/components/Forms/common/mappings.ts

```
@@ -134,220 +151,285 @@ export const toCreateTemplateMutationVariables = (
-                displayName: 'Whiteboard Template',
-              };
-            }
-            break;
-          }
-          case CalloutType.WhiteboardCollection: {
-            delete calloutDraft.callout.framing.whiteboard;
-            delete calloutDraft.callout.contributionDefaults?.postDescription;
-            break;
-          }
+        case CalloutType.LinkCollection: {
+          delete callout.contributionDefaults;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)